src/Commands/Sql/Db/DbAdviseCommand.cs

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.Models.Option;
+using AzureMcp.Models.Sql;
+using AzureMcp.Options.Sql.Database;
+using AzureMcp.Services.Azure.Sql.Exceptions;
+using AzureMcp.Services.Interfaces;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Commands.Sql.Db;
+
+public sealed class DbAdviseCommand(ILogger<DbAdviseCommand> logger)
+    : BaseDbCommand<DatabaseAdviseOptions>
+{
+    private const string _commandTitle = "Get SQL Database Advisor Recommendations";
+    private readonly ILogger<DbAdviseCommand> _logger = logger;
+
+    private readonly Option<string> _tableName = OptionDefinitions.Sql.Table;
+    private readonly Option<int> _minImpact = OptionDefinitions.Sql.MinimumImpact;
+    private readonly Option<string> _advisorType = OptionDefinitions.Sql.AdvisorType;
+
+    public override string Name => "advise";
+    public override string Title => _commandTitle; public override string Description =>
+        """
+        Optimize and improve performance of your Azure SQL database by getting intelligent advisor recommendations.
+        This command analyzes your database and provides optimization suggestions to help improve performance, reduce costs, and enhance query efficiency.
+        
+        The Azure SQL Database advisor provides recommendations for:
+        - Index optimization (create/drop suggestions to improve query performance)
+        - Query performance tuning and optimization
+        - Database parameterization for better plan reuse
+        - Force last good execution plans for regression fixes
+        - Performance bottleneck identification and resolution
+        
+        Use this command when you want to:
+        - Optimize database performance
+        - Improve slow running queries
+        - Reduce database costs through better resource utilization
+        - Get AI-powered insights for database tuning
+        - Identify and fix performance regressions
+        
+        Required options:
+        - database: The name of the database to analyze and optimize
+        - server-name: The name of the SQL server containing the database
+        Optional:
+        - table: Focus optimization recommendations on a specific table
+        - minimum-impact: Set minimum performance impact threshold for recommendations
+        - advisor-type: Filter by specific advisor type (CreateIndex, DropIndex, ForceLastGoodPlan, DbParameterization)
```

Comment:
Isn't this redundant since this information should already be present in the tool argument descriptions?